src/microbe_stage/MicrobeStage.cs

```diff
@@ -379,6 +387,8 @@ public void SpawnPlayer()
 
         Camera.ObjectToFollow = Player;
 
+        spawner.DespawnAll();
```

Comment:
Is there a chance that this crashes the game? There was that one random guy who tried to crank up the despawn rate a ton and seemed to get more crashes that way. Anyone experienced crashes in testing?

Reply:
@buckly90 @Nunez2196 @Oliveriver ?

Reply:
It *might* help a bit to use the `Invoke.Instance.Queue` to trigger this on the next frame as on this frame the player is already spawned, so that would spread things out a bit over different frames, though I'm not sure if that has any effect on anything...

Reply:
Nothing close to a crash for me.

Reply:
I could certainly add more stuff here if you want me to, but obviously with no crashes so far I can't prove that it's doing anything.

Reply:
i didn't experience any crashes either